data/funding_bodies.yml

```diff
@@ -15,6 +15,10 @@ items:
      id: "bmbf"
      link: https://www.bmbf.de/
      logo: /images/funding/bmbf.png
+  - name: Bundesministerium für Wirtschaft und Energie
```

Comment:
You need an additional space before the dash. YAML is very specific about consistent spacing, and this is causing this file to be unreadable

Reply:
ok done